src/utils/fields.js

```diff
@@ -126,6 +126,25 @@ const PREVIEW_FIELDS_TO_FETCH = [
     FIELD_IS_DOWNLOAD_AVAILABLE,
 ];
 
+// Fields needed to get versions for a file in activity feed
+const FEED_FILE_VERSIONS_FIELDS_TO_FETCH = [
+    FIELD_AUTHENTICATED_DOWNLOAD_URL, // Increases load time for the activity feed
```

Comment:
I think we want to remove these fields as well, but we should comment their use on `FILE_VERSIONS_FIELDS_TO_FETCH`.

Reply:
Ah yes, I was going to do that in a separate PR; just in case QA finds any issues that we missed with just removing the retention field. But I can include it here if that's what we want to do!

Reply:
Let's do it all together, since the changes will roll into a single BUIE release anyway.